README.md

```diff
@@ -110,9 +110,19 @@ class MyController {
   @authenticate({
     scope: ['read:users']
   })
-    async users(): Promise<string> {
-      return 'users';
-    }
+  async users(): Promise<string> {
+    return 'users';
+  }
+
+  // This route has a dynamic scope parameter for validation.
+  // It will block a JWT that does not contain the "tenantId" value received in the route path.
+  @authenticate({
+    scope: ['{path.tenantId}:read:users']
```

Comment:
is this a unique string tenant id or numeric tenant id?

Reply:
It can be whatever you want it to be.